### ISSUE TITLE
JWT token deprecation

### ISSUE THREAD

**Issue by YoussefAbdelmonem** [evilrat/flutter_zoom_sdk#126]
Zoom has deprecated jwt token since june 1 2023 and the end of using jwt was 8 September 2023 so the plugin needs to be updated to the recent changes that happen in Zoom

**Comment by WeappDevs** [comment]
Yes, I am also facing this also. Plz try to resolve it as soon as possible.
